troposphere/static/js/actions/instance/redeploy.js

```diff
@@ -23,7 +23,7 @@ define(function (require) {
         attrs: {action: "redeploy"}
       }).done(function(){
         instance.set({
-          state: new InstanceState({status_raw: "active - initializing"})
+          state: instanceState
```

Comment:
Thanks again :recycle: 

![be kind, rewind](https://media.giphy.com/media/pWsz9pgd1X1Re/giphy.gif)